packages/gatsby-source-contentful/README.md

```diff
@@ -230,23 +230,44 @@ To get **all** the `CaseStudy` nodes with ShortText fields `id`, `slug`, `title`
 It is strongly recommended that you take a look at how data flows in a real Contentful and Gatsby application to fully understand how the queries, Node.js functions and React components all come together. Check out the example site at
 [using-contentful.gatsbyjs.org](https://using-contentful.gatsbyjs.org/).
 
-## **Beta** [Contentful Rich Text](https://www.contentful.com/developers/docs/concepts/rich-text/)
+## [Contentful Rich Text](https://www.contentful.com/developers/docs/concepts/rich-text/)
 
-Rich text feature is supported in this source plugin, if you want to serialize the field content to html you can add the plugin `@contentful/gatsby-transformer-contentful-richtext`.
+Rich Text feature is supported in this source plugin, you can use the following query to get the json output:
 
-After adding the transformer plugin you can use the following query to get the html output:
-
-```
+```graphql
 {
   allContentfulBlogPost {
     bodyRichText {
       childContentfulRichText {
```

Comment:
I think `childContentfulRichText` is no longer needed here (and would actually break) because `json` should be under `bodyRichText`.